rest-service/manager_rest/test/endpoints/test_inter_deployment_dependencies.py

```diff
@@ -103,13 +111,17 @@ def test_deletes_existing_dependency(self):
             0,
             len(self.client.inter_deployment_dependencies.list())
         )
+        mock_modify_graph.assert_called_with(ANY, ANY, 'remove')
 
-    def test_fails_to_delete_non_existing_dependency(self):
+    @patch('manager_rest.resource_manager.ResourceManager'
```

Comment:
What does this decorator do?

Reply:
it's for asserting that the mocked function is called